packages/components/src/date-time/style.scss

```diff
@@ -237,6 +237,10 @@
 	}
 }
 
+.components-datetime__time-field-integer-field {
```

Comment:
Tiny nit: looks like this class is also applied to the Months dropdown in the date time picker, so maybe we should rename it to a name that doesn't refer to "integer" ?

Reply:
Good catch, I didn't even notice! I considered renaming it to "form", but looking at the code again, the "integer" in the internal component `<UpdateOnBlurAsIntegerField>` seems to be about handling integer validation concerns. So the fact that the Months dropdown is using it looks like an abstraction failure, rather than just inappropriate naming 😬 I think we should keep this as is for now, until it causes issues.